common/scala/src/main/scala/whisk/core/entity/WhiskStore.scala

```diff
@@ -169,30 +169,26 @@ object WhiskEntityQueries {
   val TOP = "\ufff0"
   val WHISKVIEW = "whisks"
   val ALL = "all"
-  val ENTITIES = "entities"
```

Comment:
@cbickel i removed the view change from this pr but eliminated the reference to the view from the core.